org.qutebrowser.qutebrowser.yml

```diff
@@ -0,0 +1,43 @@
+---
+app-id: org.qutebrowser.qutebrowser
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: qutebrowser
+rename-desktop-file: qutebrowser.desktop
```

Comment:
It would be nice if you could rename those upstream.

Reply:
I agree, especially given appdata is correctly named.

Reply:
Done.